dev-docs/specs/project-workspaces.md

```diff
@@ -0,0 +1,441 @@
+# Workspace Architecture Implementation Plan
+
+## Overview
+Refactor the storage system to use a unified workspace model where all storage is organized as workspaces containing projects. Add breadcrumb UI showing "workspace / project", improve import flow with project naming, and add workspace naming on open.
+
+## Key Design Decisions
+- **Workspace Structure**: Flat - `workspace-folder/project-name/noodles.json + data/`
+- **Built-in Workspaces**: Browser Storage (OPFS) and Examples (publicFolder) act as special workspaces
```

Comment:
Do we need opfs projects? Probably better is files loaded over the network. Or some kind of collaborative link without a directory setup